DocNet.Console/DocNet.Console.csproj

```diff
@@ -23,6 +23,7 @@
     <WarningLevel>4</WarningLevel>
     <RunCodeAnalysis>true</RunCodeAnalysis>
     <CodeAnalysisRuleSet>..\.global\DocNet.ruleset</CodeAnalysisRuleSet>
+    <NoWarn>CA1303</NoWarn>
```

Comment:
Just add a local suppression instead of disabling the warning. Makes it more visible.